packages/flow-core/src/components/f-accordion/f-accordion.ts

```diff
@@ -71,6 +74,19 @@ export class FAccordion extends FRoot {
 		this.dispatchToggleEvent(this.open);
 	}
 
+	/**
+	 * changeState
+	 */
+	stateChange(mouseState: "enter" | "leave") {
+		if (this.fAccordionHeader) {
+			if (mouseState === "enter") {
+				this.fAccordionHeader.style.background = "var(--color-surface-default-hover)";
```

Comment:
you can do

`this.fAccordionHeader.state="default"`
on leave
`this.fAccordionHeader.state="transparent"`

Reply:
since state variables are different, hence this is not required